src/node.cc

```diff
@@ -2611,6 +2611,30 @@ void SetupProcessObject(Environment* env,
                     "platform",
                     OneByteString(env->isolate(), NODE_PLATFORM));
 
+  // process.release
+  Local<Object> release = Object::New(env->isolate());
+  READONLY_PROPERTY(process, "release", release);
+
+  READONLY_PROPERTY(release, "name", OneByteString(env->isolate(), "iojs"));
+
+#if defined(NODE_RELEASE_SOURCE_URL)
+  READONLY_PROPERTY(release,
+                    "sourceUrl",
+                    OneByteString(env->isolate(), NODE_RELEASE_SOURCE_URL));
+#endif
+
+#if defined(NODE_RELEASE_HEADERS_URL)
+  READONLY_PROPERTY(release,
+                    "headersUrl",
+                    OneByteString(env->isolate(), NODE_RELEASE_HEADERS_URL));
+#endif
+
+#if defined(NODE_RELEASE_LIB_URL)
+  READONLY_PROPERTY(release,
+                    "libUrl",
+                    OneByteString(env->isolate(), NODE_RELEASE_LIB_URL));
+#endif
```

Comment:
Isn't it possible to reconstruct the download URL from the NODE_VERSION_* macros in src/node_version.h?  The approach you've taken is not bad but it's considerably more complex than is strictly necessary, I think.